cityofaustin/atd-vz-data
# 9497 create new note

## Associated issues

Closes cityofaustin/atd-data-tech#9497

## Testing
**URL to test:** 

This can be tested locally or as a Netlify preview (https://deploy-preview-1079--atd-vze-staging.netlify.app/#/dashboard)

**Steps to test:**

1. Navigate to Crashes tab and select any crash ID.
2. Test adding a new note to the Notes table.
3. Table should repopulate with your new note, your email, and the current date. The input text box should automatically have cleared.
4. Go to crash details page for crash ID #17727381 to see what the Notes table looks like when populated with multiple text entries and longer paragraphs. (https://deploy-preview-1079--atd-vze-staging.netlify.app/#/crashes/17727381)

---
#### Ship list
